holochain-introduction-for-developers/holochain-brief-introduction.md

```diff
@@ -0,0 +1,75 @@
+
+* A **peer-to-peer protocol** that shares data among participants, giving them the power to act collectively as an 'immune system' that detects invalid data and pushes out bad actors.
+* An open-source **application development framework** that abstracts away complexity and allows you to focus on your core functionality.
+* A set of **core apps and utilities**, including an identity manager, a private data vault, and a software distribution platform.
+* A **distributed hosting ecosystem** called Holo Hosting which bridges Holochain to the web so people can use your apps as if they were traditional, server-based apps.
+
+### Uncompromisable sovereignty
+
+Each participant is in control of their own identity and data. Private data stays private, and public data is cryptographically signed to prove ownership and authenticity.
+
+### Intrinsic data integrity
+
+Rather than getting everyone to agree on a single global timeline, Holochain lets data speak for itself. Every participant has a copy of an app's validation rules, which gives them the power to audit other participants' data.
+
+### Resilience through interconnection
+
+Everyone is part of a peer-to-peer network that replicates public data, audits it, and spreads warnings of bad actors. Third-party validators are chosen at random to thwart collusion.
+
+### Circles of access
+
+Every Holochain app enjoys its own private network. There is no global ledger. Groups can even fork an app for increased privacy. Individuals can choose to 'bridge' from one space to another to share data as needed.
+
+### Consensus where it matters
+
+Participants are equipped with rich information about their peers and come to their own conclusions about whom to trust. Because a coffee purchase shouldn't need the whole world's approval.
+
+### Anti-fragility
+
+Holochain networks constantly redistribute responsibility for data and traffic, which means that they flow nimbly around outages, partitions, and attacks. Apps can even work offline.
+
+### High performance, low cost
+
+Distributed apps will only succeed if they're a pleasure to use. Because Holochain favors direct peer interaction over global consensus, applications are faster, more responsive, and more respectful of limited computing resources.
+
+### Rapid application development
+
+We're building a framework that handles the ins-and-outs of peer coordination so you can focus on building the things that matter to your users. Holochain is lightweight, batteries included, and available for most operating systems.
+
+### Versatile organizational forms
+
+Holochain doesn't impose any economic, governance, licensing, or business models on your project. You're free to use Holochain's tools to build software for a commons, a non-profit, a startup, or a huge corporation.
```

Comment:
This isn't quite true.  We are working on the cryptographic automony licence, which does create some limitations.  See https://medium.com/holochain/understanding-the-cryptographic-autonomy-license-172ac920966d

Reply:
oh yeah, I guess it does. After reading everything about the CAL, I feel like I'm 90% certain about my interpretation... tl;dr is:

* it's not viral like the GPL
* you can't control someone else's keys

Is that about right @zippy?